sql/core/src/test/scala/org/apache/spark/sql/SQLQuerySuite.scala

```diff
@@ -2376,4 +2376,47 @@ class SQLQuerySuite extends QueryTest with SharedSQLContext {
           Row("r3c1x", "r3c2", "t1r3c3", "r3c2", "t1r3c3") :: Nil)
     }
   }
+
+  test("show tables") {
```

Comment:
Can you move this test case into `DDLSuite.scala`? `SQLQuerySuite` is already very big now.